hadoop/pom.xml

```diff
@@ -85,26 +85,6 @@
                     </archive>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
this plugin is for generating `hadoop-tsfile-0.11.0-SNAPSHOT-jar-with-dependencies.jar` file, which includes all dependencies for using hdfs and tsfile.
If the user uses maven, indeed it is useless, but for users who are not use maven, I think it is useful (or someone who uses maven but there are version conflicts for hadoop dependencies).

So, I have no idea about whether to remove this plugin? either is ok.

What do you think?

Reply:
Ok, if `hadoop-tsfile-0.11.0-SNAPSHOT-jar-with-dependencies.jar` is needed somewhere, i do think there is no enough reason to remove it.

Thanks for your kindly review~